web_widget_kanban_many2one/static/src/js/web_widget_kanban_many2one.js

```diff
@@ -0,0 +1,62 @@
+odoo.define('web.kanban_editable', function(require){
```

Comment:
There should be a copyright line in this file such as [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/static/src/js/web_module_name.js#L1)